### PR TITLE
Webhook Docs

### LINKED ISSUE #10: Google to Basecamp connector page
We want a page documenting how to set up and use a basecamp connector. Please include where to find all the code.

### PR DESCRIPTION
**Pull request by awtpi314**
Closes #10

### CODE REVIEW

File: Dev-Tools/basecamp-connector/tool.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+Thankfully the Google Apps Script environment has everything that you would need for making a device like this.
+There aren't too many things you have to do to set up the new project. The most important thing is making sure that you have edit access to the form.
+Without this edit access, you will not be able to create a script that will run on form submissions.
+
+The next thing to do is to link the library that we created to facilitate easier creation of these connectors.
+
+![Add library button](./images/add_cloudflare_library.png)
+
+Once the dialog opens and you see the text box, enter `1klv2AvLhl7gGeBTJjItWbnLeKS5gRcg1jWXEpmchs27gKkBxV1gvlFaG` and press add.
```
Comment (SpBills) [inline Dev-Tools/basecamp-connector/tool.md:21]: Should this be private?

File: Dev-Tools/basecamp-connector/tool.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+![Add trigger dialog](./images/trigger_details_dialog.png)
+
+Press save and then you have your project all set up. You can now start to code the actual connector.
+
+### Coding the connector
+
+Head back to the code page and then copy and paste this code into the editor. This is a template for how to set up the connector.
+
+```js
```
Comment (SpBills) [inline Dev-Tools/basecamp-connector/tool.md:45]: This code works without any imports?

**Review by SpBills — CHANGES_REQUESTED** [review]
- Leave comments on the code so that we can understand what it is all doing
- Add a link for this page on the main README.

**Comment by SebastianNeiswanger** [comment]
I worked with Spencer in reviewing and agree with his comments.

File: Dev-Tools/basecamp-connector/tool.md (hunk 1 above)
Comment (awtpi314) [reply Dev-Tools/basecamp-connector/tool.md:21]: This does not need to be private as the library is only available to people we share it with.

File: Dev-Tools/basecamp-connector/tool.md (hunk 2 above)
Comment (awtpi314) [reply Dev-Tools/basecamp-connector/tool.md:45]: Yes. AppScript takes care of all of that.

**Pull request merged**